compiler/luci-interpreter/src/kernels/Conv2D.cpp

```diff
@@ -87,6 +88,22 @@ void Conv2D::configure()
                                   filter_width, output_width);
 
   _output->resize({batches, output_height, output_width, output_depth});
+
+  // Allocate tensor for Im2Col, if needed.
+  // The checks here should be aligned with the actual implementation.
+  const bool need_dilated_im2col =
+      _params.dilation_height_factor != 1 || _params.dilation_width_factor != 1;
+  const bool need_non_dilated_im2col = _params.stride_height != 1 || _params.stride_width != 1 ||
+                                       filter_height != 1 || filter_width != 1;
+  const bool need_im2col = need_dilated_im2col || need_non_dilated_im2col;
+  if (need_im2col)
+  {
+    const int input_depth = input_shape.dim(3);
+    Shape im2col_shape{batches, output_height, output_width,
+                       input_depth * filter_height * filter_width};
+    _im2col =
+        std::make_unique<Tensor>(_filter->element_type(), im2col_shape, AffineQuantization{}, "");
```

Comment:
According tflite, we should use type of input

Reply:
Fixed, my bad.